Description: gstreamer: fix build issue with gst 1.0 as fullscreen is not yet supported

Origin: vendor
Forwarded: not-needed

diff --git a/Source/WebCore/platform/graphics/gstreamer/PlatformVideoWindowPrivate.h b/Source/WebCore/platform/graphics/gstreamer/PlatformVideoWindowPrivate.h
index f48ff83..f5d841f 100644
--- a/Source/WebCore/platform/graphics/gstreamer/PlatformVideoWindowPrivate.h
+++ b/Source/WebCore/platform/graphics/gstreamer/PlatformVideoWindowPrivate.h
@@ -25,6 +25,8 @@
 
 #include <QWindow>
 
+#if ENABLE(VIDEO) && USE(GSTREAMER) && !defined(GST_API_VERSION_1)
+
 namespace WebCore {
 
 class HTMLVideoElement;
@@ -55,5 +57,6 @@ private:
 
 } // namespace WebCore
 
+#endif /* #if ENABLE(VIDEO) && USE(GSTREAMER) && !defined(GST_API_VERSION_1) */
 
 #endif // PlatformVideoWindowPrivate_h
